packages/block-library/src/navigation/block.json

```diff
@@ -84,6 +84,10 @@
 		"templateLock": {
 			"type": [ "string", "boolean" ],
 			"enum": [ "all", "insert", "contentOnly", false ]
+		},
+		"accessibleMenuName": {
```

Comment:
Hey 👋 

Based on https://github.com/WordPress/gutenberg/pull/41744 I would think we actually want to use the `ariaLabel` name for the attribute 🤔

Reply:
@fabiankaegy rename `accessibleMenuName` to `ariaLabel`.